### DISCUSSION TITLE
Welcome to AngouriMath Discussions!

### DISCUSSION THREAD

**Discussion by WhiteBlackGoose** [asc-community/AngouriMath#312] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by valxyz** [comment]
mod (%) is defined?
Example: "17 % 5" or "mod(17,5)" or "17 mod 5"

**Reply by WhiteBlackGoose** [reply]
No, I don't think we have it

**Reply by valxyz** [reply]
Thanks for the answer, anyway AM is a cool parser! :D

**Reply by WhiteBlackGoose** [reply]
:smile:
